prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+words_to_count = {}
+lengths = []
+
+text = input("Text: ")
+words = text.split(" ")
+for text in words:
+    if text in words_to_count.keys():
+        words_to_count[text] += 1
+    else:
+        words_to_count[text] = 1
+listed_words = words_to_count.keys()
```

Comment:
listed words is defined only to be used on the next line you could instead go sorted_words(words_to_count.keys())